docs/management/connectors/action-types/inference.asciidoc

```diff
@@ -0,0 +1,123 @@
+[[inference-action-type]]
+== {infer-cap} connector and action
+++++
+<titleabbrev>{inference}</titleabbrev>
+++++
+:frontmatter-description: Add a connector that can send requests to {inference}.
+:frontmatter-tags-products: [kibana] 
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [configure]
+
+
+The {infer} connector uses {es} client to send requests to {infer}. The connector uses the <<execute-connector-api,run connector API>> to send the request.
+
+[float]
+[[define-inference-ui]]
+=== Create connectors in {kib}
+
+You can create connectors in *{stack-manage-app} > {connectors-ui}*.  For example:
+
+[role="screenshot"]
+image::management/connectors/images/inference-connector.png[{inference} connector]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+[float]
+[[inference-connector-configuration]]
+==== Connector configuration
+
+{infer-cap} connectors have the following configuration properties:
+
+Name::      The name of the connector.
+Service::   The {infer} supported provider service.
+Task type:: The {infer} task type, it depends to the selected service.
+Inference ID:: The {infer} ID.
+Provider configuration:: Settings for service configuration.
+Provider secrets:: Configuration for authentication.
+Task type configuration:: Settings for task type configuration.
+
+[float]
+[[inference-action-configuration]]
+=== Test connectors
+
+You can test connectors with the <<execute-connector-api,run connector API>> or
+as you're creating or editing the connector in {kib}. For example:
+
+[role="screenshot"]
+image::management/connectors/images/inference-completion-params.png[{infer} params test]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
```

Comment:
I think it would be better to have all the task types listed under a main section. Please review the title suggestion below (`Inference connector actions`) as I'm not entirely familiar with the connector terminology.

```suggestion
[float]
[[inference-connector-actions]]
=== {infer-cap} connector actions

```